src/mame/drivers/hp9k_3xx.cpp

```diff
@@ -146,7 +146,7 @@ class hp9k3xx_state : public driver_device
 		printf(")\n");
 #endif
 	}
-
+	void hp9k300(machine_config &config);
```

Comment:
This should be protected since it's a partial machine.

Also, the stuff above with LEDs should be done through the output system, so it can be logged with `-output console`, displayed using the artwork system, or fed to an external program.